ssf-ruby.gemspec

```diff
@@ -13,5 +13,8 @@ spec = Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- test/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ['lib']
-  s.add_runtime_dependency 'google-protobuf', ["= 3.3.0"]
+  s.add_runtime_dependency 'google-protobuf', ['= 3.3.0']
+  s.add_runtime_dependency 'minitest', ['= 5.8.4']
```

Comment:
I think `minitest` and `rake` should be `add_development_dependency` not `add_runtime_dependency`